src/main/java/frc/robot/commands/AutoIntakeCommand.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.filter.Debouncer.DebounceType;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.CrocodileSubsystem;
+
+public class AutoIntakeCommand extends CommandBase {
+    private final CrocodileSubsystem crocodileSubsystem;
+    private final double speed;
+    private final Debouncer debouncer;
+
+    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed) {
+        this.crocodileSubsystem = crocodileSubsystem;
+        this.speed = speed;
+        debouncer = new Debouncer(0.5, DebounceType.kBoth);
+        addRequirements(crocodileSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        crocodileSubsystem.setIntakeSpeed(speed);
+    }
+
+    @Override
+    public boolean isFinished() {
+        // will return true if beam break is true for 0.5 seconds
+        return debouncer.calculate(crocodileSubsystem.getBeamBreak());
```

Comment:
this will cause issues when de-intaking. Consider passing a boolean to this command that changes the isFinished behavior.

Reply:
another minor note, but 0.5 may not be the right value. This will require tuning.